vector/src/main/java/im/vector/preference/BingRulePreference.kt

```diff
@@ -149,6 +156,39 @@ class BingRulePreference : VectorPreference {
         return rule
     }
 
+    override fun onBindViewHolder(holder: PreferenceViewHolder) {
+        super.onBindViewHolder(holder)
+        val radioGroup = holder.findViewById(R.id.bingPreferenceRadioGroup) as? RadioGroup
+
+        when (ruleStatusIndex) {
+            NOTIFICATION_OFF_INDEX -> {
+                radioGroup?.check(R.id.bingPreferenceRadioBingRuleOff)
+            }
+            NOTIFICATION_ON_INDEX -> {
+                radioGroup?.check(R.id.bingPreferenceRadioBingRuleOn)
```

Comment:
`bingPreferenceRadioBingRuleOn` should be renamed

Reply:
done